src/mon/Elector.h

```diff
@@ -191,7 +191,7 @@ class Elector : public ElectionOwner, RankProvider {
   bool send_peer_ping(int peer, const utime_t *n=NULL);
   /**
    * Check the state of pinging the specified peer. This is our
-   * "tick" for heartbeating; scheduled by itself and begin_peer_ping().
+   * "tick" for heart beating; scheduled by itself and begin_peer_ping().
```

Comment:
Leave this one alone.  I submit that in Ceph's context `heartbeat` is a verb that does not contain a space.